backend/routes/api/contributors.py

```
@@ -230,42 +325,34 @@ async def get_metrics_contributors(
+             WHERE label_elem->>'name' LIKE 'sig-%'
+             LIMIT 1) AS pr_sig_label
         FROM webhooks
         WHERE event_type = 'pull_request_review'
-          AND action = 'submitted'
+          AND action != 'dismissed'
           AND sender IS DISTINCT FROM pr_author
           """
         + time_filter
         + user_filter_reviewers
         + exclude_user_filter_reviewers
         + repository_filter
```

Ruff (0.14.8)

Possible SQL injection vector through string-based query construction

(S608)